scripts/questions_gen.py

```diff
@@ -59,25 +59,27 @@ def eval_parser(eval_response: str):
     eval_response_parsed = eval_response.split("\n")
     eval_len = len(eval_response_parsed)
 
-    match eval_len:
-        # TODO: OLS-606 Incorrect value is returned from eval_parser when the
-        #               response contains no data or just one line
-        case 0:
-            return 0, eval_response_parsed
-        case 1:
-            return 0, eval_response_parsed
-        case 2:
-            score_str = eval_response_parsed[0]
-            score = float(score_str) if score_str else 0
-            reasoning = eval_response_parsed[1]
-            return score, reasoning
-        case 3:
-            score_str, reasoning_str = eval_response_parsed[1], eval_response_parsed[2]
-            score = float(score_str) if score_str else 0
-            reasoning = reasoning_str.lstrip("\n")
-            return score, reasoning
-        case _:
-            return 0, eval_response
+    no_data_lines = 0
+    single_lines = 1
+    double_lines = 2
+    triple_lines = 3
+
+    if eval_len == no_data_lines:
```

Comment:
why was match-case removed? Is not it much more readable?

Reply:
says this in the story - commented/named magic constants in eval_parser

Reply:
then just use named constants in pattern matcher ;)